app/Http/Transformers/SignupTransformer.php

```diff
@@ -34,12 +34,14 @@ class SignupTransformer extends TransformerAbstract
      */
     public function transform(Signup $signup)
     {
+        $firstPost = $signup->posts()->first();
+
         return [
             'signup_id' => $signup->id,
             'northstar_id' => $signup->northstar_id,
             'campaign_id' => $signup->campaign_id,
             'campaign_run_id' => $signup->campaign_run_id,
-            'quantity' => $signup->getQuantity(),
+            'quantity' => $firstPost ? $firstPost->getTotalQuantity() : $signup->getQuantity(),
```

Comment:
hmm I wonder if `getQuantity()` on the signup should be updated to include the logic that you do here (see if there are posts, grab the quantity from them if it can, otherwise return what it can find in the signups table)

Reply:
@katiecrane yes I like that idea! I'll update now!

Reply:
the only thing is I'm worried about is the conditionals looping back and forth between these two functions (`getQuantity` and `getTotalQuantity`). Once we move the quantity to the posts, will there still be a `quantity` or `quantity_pending` on the signup?

Reply:
ah if you add all the logic into `getQuantity()` then you can just call that here without ever calling `getTotalQuantity()`. Now I'm thinking it should have been that initially instead of the function on the post model.

Reply:
Initially there will be some quantities that exist in both places, but we should only need to worry about the column on the post table at that point I think

Reply:
ahhh I see that makes sense! Thank you @katiecrane ! Combined `getTotalQuantity` into `getQuantity` - does this still work for the reportback transformer?

Reply:
@chloealee yes I think the reportback transformer is great with this!